evalml/tests/component_tests/test_utils.py

```diff
@@ -92,15 +92,51 @@
         "XGBoost Regressor",
     ]
 )
+not_supported_in_conda = set(
+    [
+        "Prophet Regressor",
+    ]
+)
+not_supported_in_windows = set(
+    [
+        "Prophet Regressor",
+    ]
+)
+not_supported_in_windows_py39 = set(
+    [
+        "Prophet Regressor",
+    ]
+)
+
 # Keeping here in case we need to add to it when a new component is added
 not_supported_in_linux_py39 = set()
-not_supported_in_conda = set()
-not_supported_in_windows = set()
-not_supported_in_windows_py39 = set()
 
 
-def test_all_components():
-    expected_components = all_requirements_set
+def test_all_components(
+    is_running_py_39_or_above,
+    is_using_conda,
+    is_using_windows,
+):
+    if is_using_conda:
+        # No prophet, ARIMA, and vowpalwabbit
+        expected_components = all_requirements_set.difference(not_supported_in_conda)
+
+    elif is_using_windows and not is_running_py_39_or_above:
```

Comment:
if the issue was conda - can we test dropping these windows + py 39 cases?

Reply:
Probably! I did it earlier but wasn't sure if that was the right approach. I'll change it.

Reply:
if the purpose of your previous PR was to increase coverage for py39 (and these test cases still pass under those scenarios) I would say to drop these cases but keep the conda case because that is still needed. Thank you!